test/solc/CommandLineParser.cpp

```diff
@@ -422,6 +420,60 @@ BOOST_AUTO_TEST_CASE(standard_json_mode_options)
 	BOOST_TEST(parsedOptions.value() == expectedOptions);
 }
 
+BOOST_AUTO_TEST_CASE(invalid_combinations)
+{
+	for (InputMode inputMode: {InputMode::StandardJson, InputMode::Assembler})
+	{
+		vector<string> commandLine = {
+			"solc",
+			"--experimental-via-ir",
+			"input.json",
+		};
+
+		if (inputMode == InputMode::StandardJson)
+			commandLine += vector<string>{
+				"--standard-json",
+			};
```

Comment:
If it's just a single parameter, I'd shorten it to:
```suggestion
			commandLine.push_back("--standard-json");
```

Reply:
This is fixed